pkg/controller/migmigration/annotation.go

```diff
@@ -318,21 +363,26 @@ func (t *Task) annotatePVs(client k8sclient.Client) error {
 		// Update
 		err = client.Update(context.TODO(), &pvcResource)
 		if err != nil {
-			return liberr.Wrap(err)
+			return itemsUpdated, liberr.Wrap(err)
 		}
 		log.Info(
 			"PVC annotations/labels added.",
 			"ns",
 			pv.PVC.Namespace,
 			"name",
 			pv.PVC.Name)
+		itemsUpdated++
+		if itemsUpdated > 50 {
```

Comment:
Last thing I'd recommend is making this number into a const so that we can change it from one place.